lightning/src/chain/package.rs

```diff
@@ -1042,13 +1055,35 @@ mod tests {
 	fn test_package_weight() {
 		let txid = Txid::from_hex("c2d4449afa8d26140898dd54d3390b057ba2a5afcf03ba29d7dc0d8b9ffe966e").unwrap();
 		let secp_ctx = Secp256k1::new();
-		let revk_outp = dumb_revk_output!(secp_ctx);
 
-		let package = PackageTemplate::build_package(txid, 0, revk_outp, 0, true, 100);
-		// (nVersion (4) + nLocktime (4) + count_tx_in (1) + prevout (36) + sequence (4) + script_length (1) + count_tx_out (1) + value (8) + var_int (1)) * WITNESS_SCALE_FACTOR
-		// + witness marker (2) + WEIGHT_REVOKED_OUTPUT
-		for &opt_anchors in [false, true].iter() {
-			assert_eq!(package.package_weight(&Script::new(), opt_anchors), (4 + 4 + 1 + 36 + 4 + 1 + 1 + 8 + 1) * WITNESS_SCALE_FACTOR + 2 + WEIGHT_REVOKED_OUTPUT as usize);
+		{
+			let revk_outp = dumb_revk_output!(secp_ctx);
+			let package = PackageTemplate::build_package(txid, 0, revk_outp, 0, true, 100);
+			// (nVersion (4) + nLocktime (4) + count_tx_in (1) + prevout (36) + sequence (4) + script_length (1) + count_tx_out (1) + value (8) + var_int (1)) * WITNESS_SCALE_FACTOR
+			// + witness marker (2) + WEIGHT_REVOKED_OUTPUT
+			for &opt_anchors in [false, true].iter() {
+				assert_eq!(package.package_weight(&Script::new(), opt_anchors), (4 + 4 + 1 + 36 + 4 + 1 + 1 + 8 + 1) * WITNESS_SCALE_FACTOR + 2 + WEIGHT_REVOKED_OUTPUT as usize);
```

Comment:
DRY `(4 + 4 + 1 + 36 + 4 + 1 + 1 + 8 + 1)` into a constant, and also the comment?

Reply:
done